packages/charts/src/components/PieChart/PieChart.tsx

```diff
@@ -143,6 +145,85 @@ const PieChart: FC<PieChartProps> = forwardRef((props: PieChartProps, ref: Ref<H
     [onDataPointClick]
   );
 
+  const renderActiveShape = useCallback(
+    (props) => {
```

Comment:
this code is copied from recharts, right? we have to add the corresponding license entry to the .reuse/dep5 file for this file

Reply:
compare e.g with the web components project: https://github.com/SAP/ui5-webcomponents/blob/master/.reuse/dep5